fdbserver/DataDistribution.actor.cpp

```diff
@@ -1690,6 +1719,20 @@ ACTOR Future<Void> bulkLoadJobDispatcher(Reference<DataDistributor> self,
 					wait(delay(0.1));
 					continue;
 				}
+
+				if (existJob.getPhase() == BulkLoadJobPhase::Error) {
+					TraceEvent(SevWarnAlways, "DDBulkLoadJobExecutorTaskError", self->ddId)
+					    .setMaxEventLength(-1)
+					    .setMaxFieldLength(-1)
+					    .detail("JobId", jobId)
+					    .detail("JobRange", jobRange)
+					    .detail("JobRoot", jobRoot)
+					    .detail("JobTransportMethod", jobTransportMethod)
+					    .detail("ExistJob", existJob.toString());
+					wait(delay(0.1));
```

Comment:
why wait?

Reply:
Avoid busy loop. We do not want bulkload take many CPU cycles in a short period